client/views/windows/help.tpl

```diff
@@ -13,11 +13,7 @@
 	</h2>
 
 	<div class="about">
-		{{#unless public}}
-			<p id="changelog-version-container" class="changelog-version">
-				Checking for updates...
-			</p>
-		{{/unless}}
+		<div id="version-checker" />
```

Comment:
Write out `</div>` please. We don't do this anywhere else.

Reply:
Sure. It's perfectly valid so I don't think we should restrict ourselves though (interestingly, that's extremely common to see that in React-land, not sure about Vue).